MapboxCoreNavigation/MMEEventsManager.swift

```diff
@@ -111,6 +116,13 @@ struct EventDetails {
             totalTimeInBackground += abs(session.lastTimeInBackground.timeIntervalSinceNow)
         }
         percentTimeInForeground = totalTimeInPortrait + totalTimeInLandscape == 0 ? 100 : Int((totalTimeInPortrait / (totalTimeInPortrait + totalTimeInLandscape) * 100))
+        currentLegIndex = routeController.routeProgress.legIndex
+        totalLegCount = routeController.routeProgress.route.legs.count
+        currentStepIndex = routeController.routeProgress.currentLegProgress.stepIndex
+        totalStepCount = routeController.routeProgress.currentLeg.steps.count
```

Comment:
Let's change these keys to:
- `stepIndex` (index of current step on current leg)
- `stepCount` (count of steps on the current leg)
- `legIndex` (index of leg in current route)
- `legCount` (count of legs on the current route)
- `totalStepCount` (sum of all steps in the route)

Reply:
Noting we already have a `stepCount` variable.